src/tests/functional/plugin/gpu/shared_tests_instances/single_layer_tests/prior_box_clustered.cpp

```diff
@@ -61,7 +61,7 @@ const auto layerSpeficParams = ::testing::Combine(
     ::testing::ValuesIn(variances)
 );
 
-INSTANTIATE_TEST_SUITE_P(smoke_PriorBoxClustered_Basic, PriorBoxClusteredLayerTest,
+INSTANTIATE_TEST_SUITE_P(smoke_PriorBoxClustered_Basic_GPU, PriorBoxClusteredLayerTest,
```

Comment:
I don't think that we need to add "_GPU" in test name prefix such as "smoke_XXX". It is required when we create GPU specific single layer test class not using test classes from share_test_class.